### ISSUE TITLE
Blinking cursor  is not visible when using dark backgrounds for Windows Binary of Zim 0.75.2 “Distraction Free Editing” plugin.

### ISSUE THREAD

**Issue by ted-kal** [zim-desktop-wiki/zim-desktop-wiki#2574]
**Description**

The following issue applies to the Windows binary. When using _Zim 0.75.2 “Distraction Free Editing”_ plugin, the blinking cursor  is not visible when using dark backgrounds. It appears that when using this mode, the cursor color is black (irrespective of what the text color is set to). So if you configure the plugin to a black background, you won’t be able to see where the cursor is located on the screen. In addition, when you have the plugin configured for a dark background, you won’t be able to see some of the menu bar buttons as they are also black in color.

**To Reproduce This Behavior on Windows**

1. Go to Edit /  Preferences / Plugins / Distraction Free Editor / Configure
2. Set text background color to black.
3. Set text foreground color to white.
4. Click 'ok' and exit the preferences window.
5. Hit *F11* to enter the distraction free editor.

**Expected behavior**

I expected that the blinking cursor would be the same color as the text foreground color in the distraction free mode, similar to the behavior of the Zim Windows binary in versions 0.6x.

**Environment**

-  Using WIndows 10 (22H2)
 - Zim Version 0.75.2

**Comment by introt** [comment]
Fix: setting `caret-color` to text color, see https://stackoverflow.com/questions/77597418/setting-cursor-color-in-gtk-3-since-3-16 

This can be done by the user by using custom CSS (perhaps via the textview prefs plugin #2576), or automatically using the following patch:

```diff
diff --git a/zim/plugins/distractionfree.py b/zim/plugins/distractionfree.py
index f0a3c938..629fc5ce 100644
--- a/zim/plugins/distractionfree.py
+++ b/zim/plugins/distractionfree.py
@@ -62,6 +62,7 @@ class DistractionFreeMainWindowExtension(MainWindowExtension):
                css = '''
                #zim-pageview text {
                        color: %(textcolor)s;
+                       caret-color : %(textcolor)s;
                        background-color: %(basecolor)s;
                }
                #zim-window-main-box scrolledwindow{
```

Problem: the default doesn't appear to be pure black, resulting in a regression in the default case.
